runtime/lua/vim/lsp/handlers.lua

```diff
@@ -33,7 +33,7 @@ local function progress_handler(_, result, ctx, _)
   local val = result.value    -- unspecified yet
   local token = result.token  -- string or number
 
-
+  if type(val) ~= 'table' then val = {val} end
```

Comment:
```suggestion
  if type(val) ~= 'table' then val = {content = val} end
```

Otherwise it is a list(?)

Reply:
…I have no idea why I didn't do that in the first place. Thanks for the hand-holding. :sweat_smile: